miniml/compiler/README.md

```diff
@@ -0,0 +1,16 @@
+# miniml compiler
+
+The file `compile.scm` is a single-file compiler for the miniml language, written in Scheme. It targets the OCaml bytecode for the version 4.07.0 of the OCaml compiler.
```

Comment:
Is it 4.07.0? I use 4.08.0's ocamlrun to run the bytecode executable.

Reply:
It's indeed 4.07.0 now, after cf4506e6561f164b1dfc1fa946141157455d6f74